src/components/AudioTrack/AudioThumb.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <!-- Should have a fixed width -->
+  <div>
+    <div class="h-0 w-full pt-full bg-yellow">&nbsp;</div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'AudioThumb',
```

Comment:
Where does the name "Thumb" come from? I've only ever seen thumb used to describe the knob on a slider before so this is new to me.

Reply:
Short for thumbnail?

Reply:
Yes, it was short for thumbnail. But I can understand the confusion so I've expanded that in bdb2f1c.